bert_sklearn/sklearn.py

```diff
@@ -155,7 +155,7 @@ def __init__(self, bert_model='bert-base-uncased',
                  gradient_accumulation_steps=1, fp16=False, loss_scale=0,
                  local_rank=-1, use_cuda=True, random_state=42,
                  validation_fraction=0.1, logfile='bert_sklearn.log',
-                 ignore_label=None):
+                 ignore_label=None, no_stdout=True, progress_bar=False):
```

Comment:
I'd probably swap the default so that the default behavior matches what it was before. Should make it easier to get merged upstream

Reply:
Oh, and add them to the docstring

Reply:
will do!